knpu/episode3/override.md

```diff
@@ -13,9 +13,12 @@ and the rebels.
 
 So let's do that, `if ($shipData['team'] == 'rebel')` which is the key inside the 
 database. Then we'll have `$ship = new RebelShip($shipData['name']);`. Else, we'll
-throw in our normal ship line, which represents the Empire ship. Ok, this doesn't
-have anything to do with Object Oriented coding, it's just a nice example of a use
-case for multiple classes. We have a database table, and you can create different
+throw in our normal ship line, which represents the Empire ship:
+
+[[[ code('3b403bbee6') ]]]
```

Comment:
I shortened this block, because the setter calls didn't change, and aren't important here
<img width="562" alt="screen shot 2015-09-27 at 7 26 52 pm" src="https://cloud.githubusercontent.com/assets/121003/10125737/c369f692-654d-11e5-9f6b-475a18c00ae7.png">